adoc/Release-Notes.adoc

```diff
@@ -33,6 +33,174 @@ endif::[]
+* Switched over to `PXC` from `cf-mysql` for database management
+* Includes these {cf} component versions:
+** app-autoscaler: 1.2.1
+** bits-service: 2.28.0
+** bpm: 1.1.0
+** capi: 1.83.0
+** cf-deployment: 9.5
+** cf-mysql: 36.15.0
+** cf-routing: 0.188.0
+** cf-sle12: 1.81.26
+** cf-sle15: 10.28
+** cf-smoke-tests: 40.0.112
+** cf-syslog-drain: 10.2
+** cf-usb: 1.0.1
+** cflinuxfs3: 0.118.0
+** credhub: 2.4.0
+** diego: 2.34.0
+** eirini: 0.0.14
+** garden-runc: 1.19.3
+** groot-btrfs: 1.0.5
+** log-cache: 2.2.2
+** loggregator: 105.5
+** loggregator-agent: 3.9
+** mapfs: 1.1.0
+** nats: 27
+** nfs-volume: 1.5.2
+** postgres-release: 26
+** pxc: 0.18.0
+** scf-helper: 1.0.3
+** statsd-injector: 1.10.0
+** uaa: 72.0
+* Buildpacks:
+** binary-buildpack: 1.0.33
+** dotnet-core-buildpack: 2.2.13
+** go-buildpack: 1.8.42
+** java-buildpack: 4.20.0 
+** nginx-buildpack: 1.0.15
+** nodejs-buildpack: 1.6.53
+** php-buildpack: 4.3.80
+** python-buildpack: 1.6.36
+** ruby-buildpack: 1.7.42
+** staticfile-buildpack: 1.4.43
+
+[id='sec.1_5.issue']
+==== Known Issues
+
+IMPORTANT: In order to deploy this version in HA, or upgrade from {cap} 1.4.1 from HA, you will need to scale down the `mysql` role to 1 in order to migrate from cf-mysql to PXC. This is based on upstream instructions but based on what we've seen with other components that rely on the database, such as Credhub, scaling all database roles into single availability helps with a stable migration and deployment.
+
+Steps for a fresh install of CAP 1.5 HA: 
+
+[arabic]
+.. Install HA UAA but scale the `mysql` role to 1 as a transition step. (In the commands below, `susecf-uaa` and `susecf-scf` are assumed to be the release names and `suse/uaa` and `suse/cf` are the chart names in the repository. Adjust those names accordingly to suit your configuration.)
++
+[source,bash]
+----
+helm install --name susecf-uaa --namespace uaa suse/uaa -f <values.yaml> --set config.HA=true \
+--set config.HA_strict=false --set sizing.mysql.count=1
+----
++
+.. Set the value of the `secrets.UAA_CA_CERT` to pass your `uaa` secret and certificate to `scf` as per Section 5.10 of the {CAP} Deployment Guide: https://www.suse.com/documentation/cloud-application-platform-1/singlehtml/book-cap-guides/book-cap-guides.html#sec-cap-install-scf-prod
++
+[source,bash]
+----
+helm install --name susecf-scf --namespace scf suse/cf -f <values.yaml> --set config.HA=true \ 
+--set config.HA_strict=false --set sizing.mysql.count=1 --set "secrets.UAA_CA_CERT=${CA_CERT}"
+----
++
+.. Scale the `mysql` role up to the default HA configuration.
++
+[source,bash]
+----
+helm upgrade susecf-uaa --namespace uaa suse/uaa -f <values.yaml> --set config.HA_strict=true \ 
+--set config.HA=true
+helm upgrade susecf-scf --namespace scf suse/cf -f <values.yaml> --set config.HA_strict=true \ 
+--set config.HA=true --set "secrets.UAA_CA_CERT=${CA_CERT}"
+----
+Steps to upgrade from CAP 1.4.1 in HA mode:
```

Comment:
May need an extra preceding blankline here to not indent on the same level as previous text (at least github renders it as part of the previous point without a blank line

Reply:
AsciiDocs isn't too particular with this as opposed to Markdown but adjusted.